lib/services/networkManagement2/package.json

```diff
@@ -6,7 +6,7 @@
     "Rajendran, Deepak <derajen@microsoft.com>",
     "Choubey, Ankur <anchoube@microsoft.com>"
   ],
-  "version": "1.3.0-preview",
+  "version": "1.4.0-preview",
```

Comment:
this describes that the changes are "minor" and non-breaking. Is that really the case? asking because you seem to have api-version update.